.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Packer
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  packer:
+    runs-on: ubuntu-latest
```

Comment:
Devemos sempre usar uma versão especifica quando possível.

https://docs.github.com/pt/actions/reference/workflow-syntax-for-github-actions

Nesse caso o valor deve ser "ubuntu-20.04"